src/test/codegen/issue-37945.rs

```diff
@@ -15,7 +15,7 @@ use std::slice::Iter;
 pub fn is_empty_1(xs: Iter<f32>) -> bool {
 // CHECK-LABEL: @is_empty_1(
 // CHECK-NEXT:  start:
-// CHECK-NEXT:    [[A:%.*]] = icmp ne {{i32\*|ptr}} %xs.1, null
+// CHECK-NEXT:    [[A:%.*]] = icmp ne {{i32\*|ptr}} {{%xs.0|%xs.1}}, null
```

Comment:
Do you know why we need to match both `%xs.0` and `%xs.1` here?

Reply:
I don't know for sure but I assume it changed the order of the slice::Iter fields. And since the field order isn't guaranteed and not the point of this test  I didn't want to just add the new value and made it work with either instead.